CMakeLists.txt

```diff
@@ -103,6 +107,43 @@ if("${_compiler_arch}" STREQUAL "")
     set(_compiler_arch ${CMAKE_SYSTEM_PROCESSOR})
 endif()
 
+# LLVM component setup
+message("Compiler architecture is ${_compiler_arch}")
+
+set(LLVM_COMPONENTS core demangle objcarcopts orcjit)
+
+if(${_compiler_arch} MATCHES "x64")
+    list(APPEND LLVM_COMPONENTS
+        x86asmparser
+        x86codegen
+        x86desc
+        x86info
+    )
+elseif(${_compiler_arch} MATCHES "aarch64")
```

Comment:
Per what @jemc said, does this need to match against arm64 too, or is that a Apple-specific naming convention?

Reply:
Yeah, it needs to match `arm64`; I'll fix it.